string map_name
int32 mode
---
std_msgs/Bool successful
